pandas/tests/tseries/offsets/conftest.py

```diff
@@ -2,6 +2,49 @@
 
 import pandas.tseries.offsets as offsets
 
+DATE_OFFSETS = {
+    "Day",
+    "MonthBegin",
+    "MonthEnd",
+    "SemiMonthBegin",
+    "SemiMonthEnd",
+    "YearBegin",
+    "YearEnd",
+    "QuarterBegin",
+    "QuarterEnd",
+    "LastWeekOfMonth",
+    "Week",
+    "WeekOfMonth",
+    "Easter",
+    "Hour",
+    "Minute",
+    "Second",
+    "Milli",
+    "Micro",
+    "Nano",
+    "DateOffset",
+}
+
+
+BUSINESS_OFFSETS = {
+    "BusinessDay",
+    "BDay",
+    "CustomBusinessDay",
+    "CBMonthBegin",
+    "CBMonthEnd",
+    "BMonthBegin",
+    "BMonthEnd",
+    "BusinessHour",
+    "CustomBusinessHour",
+    "BYearBegin",
+    "BYearEnd",
+    "CDay",
+    "BQuarterBegin",
+    "BQuarterEnd",
+    "FY5253Quarter",
+    "FY5253",
+}
+
```

Comment:
can you define offset_types in terms of the lists above?

Reply:
Sorry but I'm off due to my exams. I'll be back wihin a few days :wink: